modules/cli/src/main/scala/scala/cli/commands/SharedJvmOptions.scala

```diff
@@ -47,7 +47,13 @@ final case class SharedJvmOptions(
   @HelpMessage("Javac options")
   @Name("javacOpt")
   @Hidden
-    javacOption: List[String] = Nil
+    javacOption: List[String] = Nil,
+
+  @Group("Java")
+  @HelpMessage("Port for BSP debugging")
+  @Name("bspDebugPort")
```

Comment:
Seems this makes the duplicated option check fail (this adds the same name as the variable).
```suggestion
```

Reply:
Good point, I was copy-pasting :)